test/functional/kaui/admin_tenants_controller_test.rb

```diff
@@ -98,6 +98,115 @@ class Kaui::AdminTenantsControllerTest < Kaui::FunctionalTestHelper
     assert_equal 'Config for plugin was successfully uploaded', flash[:notice]
   end
 
+  test 'should get new catalog' do
+    tenant = create_kaui_tenant
+
+    get :new_catalog, :id => tenant.id
+    assert_response :success
+    assert has_input_field('simple_plan_product_name')
+  end
+
+  test 'should get new plan currency' do
+    tenant = create_kaui_tenant
+    plain_id = 'sailboat12345678910'
```

Comment:
Interesting: the screen seems to work with any plan id. That's probably a bug. I've created https://github.com/killbill/killbill-admin-ui/issues/114 (backlog).